hawtio-system/src/main/java/io/hawt/web/FileUploadFilter.java

```diff
@@ -0,0 +1,171 @@
+package io.hawt.web;
+
+import org.apache.commons.io.FileUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static java.lang.StrictMath.toIntExact;
+
+public class FileUploadFilter {
+
+    private static final transient Logger LOG = LoggerFactory.getLogger(FileUploadFilter.class);
+    public static final String FILE_UPLOAD_PROPNAME = "hawtio.config.fileUpload";
```

Comment:
Looks like `hawtio.config.` is for hawtio-config parameters.
http://hawt.io/configuration/index.html
Probably we should use `hawtio.upload.filter` or something?

Reply:
Alright changed to `hawtio.upload.filter`